### PR TITLE
Fix For Large Upload Playbook

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11774

## Description
The 'set' automation is used in some tasks, which makes the playbook stuck if no results are found or if integration is not available.

### CODE REVIEW

File: Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml [hunk 1]
```diff
@@ -3190,17 +3199,7 @@ tasks:
...
-                    ignorecase: true
-                  - operator: isEqualString
-                    left:
-                      value:
-                        simple: PaloAltoNetworksXDR.RiskyHost.id
-                      iscontext: true
-                    right:
-                      value:
-                        simple: PaloAltoNetworksXDR.OriginalAlert._all_events.agent_hostname
+                        simple: inputs.SrcHostname
```
Comment (ssokolovich) [inline Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml:3211]: didn't see this as a input

File: Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml [hunk 2]
```diff
@@ -3168,7 +3177,7 @@ tasks:
                       iscontext: true
                     right:
                       value:
-                        simple: PaloAltoNetworksXDR.OriginalAlert._all_events.causality_actor_primary_normalized_user.username
+                        simple: inputs.Username
```
Comment (ssokolovich) [inline Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml:3189]: didn't see this as a input
same as here

File: Packs/Core/ReleaseNotes/3_0_62.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Large Upload Alert
+
+- Updated tasks that used the **'Set'** automation to use the **'SetAndHandleEmpty'** automation.
```
[the remark this answers is not in the document]
Comment (TalNos) [reply Packs/Core/ReleaseNotes/3_0_62.md:6]: done

File: Packs/CortexXDR/ReleaseNotes/6_1_71.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex XDR - Large Upload
+
+- Updated tasks that used the **'Set'** automation to use the **'SetAndHandleEmpty'** automation.
```
[the remark this answers is not in the document]
Comment (TalNos) [reply Packs/CortexXDR/ReleaseNotes/6_1_71.md:6]: done

File: Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml [hunk 5]
```diff
@@ -2395,7 +2398,7 @@ tasks:
                     flags: {}
                     groups:
                       value:
-                        simple: "1"
+                        simple: "2"
```
[the remark this answers is not in the document]
Comment (TalNos) [reply Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml:2410]: this change was made by the 'format' command. Reverted.

File: Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml (hunk 2 above)
Comment (TalNos) [reply Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml:3189]: this change was made by the 'format' command. Reverted.

File: Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml (hunk 1 above)
Comment (TalNos) [reply Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml:3211]: Reverted

File: Packs/CortexXDR/ReleaseNotes/6_1_71.md
[the remark this answers is not in the document]
Comment (TalNos) [reply Packs/CortexXDR/ReleaseNotes/6_1_71.md:1]: reverted

File: Packs/Core/Playbooks/Playbook-Large_Upload_Alert.yml [hunk 6]
```diff
@@ -3891,6 +3897,13 @@ view: |-
     }
   }
 inputs:
+- key: InternalIPRanges
```
Comment (ssokolovich) [inline Packs/Core/Playbooks/Playbook-Large_Upload_Alert.yml:3900]: duplicated if I'm not mistaken

Comment (TalNos) [reply Packs/Core/Playbooks/Playbook-Large_Upload_Alert.yml]: fixed

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**